### DISCUSSION TITLE
2011/09/20/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95%E7%9A%84%E8%AF%AD%E9%9F%B3%E8%BE%93%E5%85%A5

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#345] [Announcements]
# 2011/09/20/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95%E7%9A%84%E8%AF%AD%E9%9F%B3%E8%BE%93%E5%85%A5

今天 偶然看到谷歌拼音输入又更新了，新版本的谷歌输入法带有语音输入功能。这个功能还是非常有用的，毕竟说话的速度远高于打字。我赶紧下载下来试使用了一下，这篇博客就是我说出来的。试用之后的第一感觉是速度太慢，说一句话要等半天才能够翻译成文字，尤其是在网络不好的情况下。目前使用语音输入的速度还不如我打字快。

https://blog.qizhen.xyz/2011/09/20/%E8%B0%B7%E6%AD%8C%E6%8B%BC%E9%9F%B3%E8%BE%93%E5%85%A5%E6%B3%95%E7%9A%84%E8%AF%AD%E9%9F%B3%E8%BE%93%E5%85%A5

**Comment by ruanqizhen** [comment]
之前博客下的留言：

qiaoyiwei
2011年09月29日 16:38 编辑
给霍金等手脚不灵便的人开发的功能，你用它干嘛。

回复
sfdb123
2011年09月21日 08:38 编辑
你试试讯飞语音输入法，我想这个应该是国人的骄傲！我的识别率100%，不管你信不信，反正我信！

回复
ruanqizhen
2011年09月21日 09:45 编辑
你推荐这个输入法只有手机端的，对我来说没有用啊。

回复
Colin (@collinzhang)
2011年09月20日 20:48 编辑
语音识别还是瓶颈。

回复
